src/scheduled-query/azext_scheduled_query/_params.py

```diff
@@ -36,3 +38,6 @@ def load_arguments(self, _):
                    options_list=['--mute-actions-duration', '--mad'],
                    help='Mute actions for the chosen period of time (in ISO 8601 duration format) after the alert is fired.')
         c.argument('actions', options_list=['--action', '-a'], action=ScheduleQueryAddAction, nargs='+', validator=get_action_group_validator('actions'))
+        c.argument('disable_auto_mitigate', arg_type=get_three_state_flag(), help='The flag that indicates whether to disable the alert automatically resolved or not.')
```

Comment:
From swagger, it's defined as `auto-mitigate` with default value `true`, could you please explain the purpose that define it as `disable-auto-mitigate`